backend/app/adapter/sqldb/migration/29_create_incident_table.sql

```diff
@@ -0,0 +1,10 @@
+-- +migrate Up
+CREATE TABLE incident
+(
+    id          CHARACTER VARYING(4)      NOT NULL UNIQUE,
+    title       CHARACTER VARYING(100)    NOT NULL UNIQUE,
```

Comment:
```suggestion
    title       VARCHAR(150)    NOT NULL,
```
Title doesn't have to be unique.